### ISSUE TITLE
ERR_CERT_COMMON_NAME_INVALID

### ISSUE THREAD

**Issue by 9WW7XxHy** [monkeyWie/proxyee#210]
![image](https://user-images.githubusercontent.com/59128273/161782245-deba8f82-11e8-44cf-ab48-7ab94e5d4686.png)

i use origin cert but it not working . Please check and fix help me . thanks all

**Comment by 9WW7XxHy** [comment]
and i had import CA bundle to system

**Comment by 9WW7XxHy** [comment]
![image](https://user-images.githubusercontent.com/59128273/161790754-daf3a256-564e-4311-9e6b-99f60ac72aff.png)

**Comment by monkeyWie** [comment]
Please show your device type and os version.

**Comment by 9WW7XxHy** [comment]
ROM PixelExperience - Android 10
Set up proxy using proxydroid and socksdroid app
i think it can't read the hostname. You can try 

and can you support more socks5 server instead of proxy http only or not

**Comment by 9WW7XxHy** [comment]
I'm building a proxy server on android and use another phone to test it

**Comment by monkeyWie** [comment]
Did you test it successfully on pc? If it successed,i think the problem might be incorrect import root ca .

**Comment by 9WW7XxHy** [comment]
Yes it works well on pc, .Same of certificates and root ca i use it very well on fiddler . But your source it error
Maybe my packet traffic is wrong format when using 2 apps sockdroid and proxydroid to set up , therefore it can't work when using traffic redirect mode over iptables 
**The strange thing is it works fine when set proxy over android wifi settings** @@
if you can't fix this problem , please support socks5 and tranparent proxy

**Comment by monkeyWie** [comment]
Actually android tun model is work well from other user's feedback, you can try use `postern`, and can you explain how your certificate was generated? This looks like a certificate generation error

**Comment by 9WW7XxHy** [comment]
> Actually android tun model is work well from other user's feedback, you can try use `postern`, and can you explain how your certificate was generated? This looks like a certificate generation error

my certificate has no problem, even if i use your certificate it still doesn't work. 
. You can try use 2 phones to check . Use sockdroid and proxydroid apps to set proxy . I'm sure this is a transmission error, so please support socks5 server and proxy transparent server, instead of just supporting http. Thanks you

**Comment by 9WW7XxHy** [comment]
> Actually android tun model is work well from other user's feedback, you can try use `postern`, and can you explain how your certificate was generated? This looks like a certificate generation error

"Actually android tun model is work well from other user's feedback"
 yes it work fine when set proxy over settings wifi 
 like that
 
![image](https://user-images.githubusercontent.com/59128273/162054481-07438bd6-97fc-4e68-afa4-b4922f065e0e.png)

 **### but i need use sockdroid and proxydroid to bypass proxy detect**
